src/main.rs

```diff
@@ -5,8 +5,10 @@
 #![allow(unused_imports)]
```

Comment:
we might put `main.rs` under a `bin` folder and name is server.rs or something

```
cargo run --bin server
```

Reply:
I think we'll probably want to split this file up a little better. I had a commit to do that, but it has a bunch of conflicts, so I'll take a crack at it again after merging.